corehq/apps/reports/static/reports/js/filters/main.js

```diff
@@ -164,6 +164,31 @@ hqDefine("reports/js/filters/main", [
             $el.koApplyBindings(model);
             model.init();
         });
+        $('.report-filter-logtag').each(function(i, el) {
+            var $el = $(el), data = $el.data();
+            if (data.defaultOn) {
+                $el.attr("name", "");
+                $el.change(function () {
+                    $el.attr("name", "logtag");
+                });
+            }
+            if (data.errorOnly) {
+                $("#device-log-errors-only-checkbox").change(function () {
```

Comment:
I had imagined this id being a data attribute, since it's passed from python along with `default_on` and `errors_only`, though I"m not sure it matters much either way.